### ISSUE TITLE
Prepare sbt-github-actions for release under sbt org

### ISSUE THREAD

**Issue by mdedetrich** [sbt/sbt-github-actions#132]
We should look into preparing/testing sbt-github-actions so it can be released under the sbt org. A few people (including myself) are waiting for this feature https://github.com/sbt/sbt-github-actions/pull/105 so this is a good testing bed to make the first sbt org release of this project.

@eed3si9n Thoughts?

**Comment by eed3si9n** [comment]
You mean release under `com.github.sbt`?

I am generally neutral about whether a plugin should publish under `com.github.sbt` or some other organization name.

**Comment by mdedetrich** [comment]
> You mean release under com.github.sbt?

I guess?

I mean whatever org sbt software is typically published under. Not sure if this is maven /sonatype sbt org or another.

**Comment by mdedetrich** [comment]
Will do, thanks for the heads up!

**Comment by mdedetrich** [comment]
@eed3si9n Sorry to ping you but because of https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/ (which was resolved in https://github.com/sbt/sbt-github-actions/pull/133) we should probably look into making release sooner rather than later.

If you don't have time can you point me to a sample sbt project as a reference and/or add in the relevant secrets to this repo and I can have a look into doing this.

**Comment by eed3si9n** [comment]
Here's my PR for setting up sbt-ci-release - https://github.com/sbt/sbt-github-actions/pull/134

**Comment by eed3si9n** [comment]
Two problems:

```
[info] welcome to sbt 1.8.2 (Eclipse Adoptium Java 11.0.17)
[info] loading settings for project sbtGithubActionsSources from build.sbt,libraries.sbt ...
[info] loading project definition from /home/runner/work/sbt-github-actions/sbt-github-actions/project/project
[info] compiling [16](https://github.com/sbt/sbt-github-actions/actions/runs/3900050793/jobs/6660371077#step:8:17) Scala sources to /home/runner/work/sbt-github-actions/sbt-github-actions/project/project/target/scala-2.12/sbt-1.0/classes ...
[warn] one feature warning; re-run with -feature for details
[warn] one warning found
[info] done compiling
[info] loading settings for project sbt-github-actions-build from libraries.sbt,plugins.sbt ...
[info] loading project definition from /home/runner/work/sbt-github-actions/sbt-github-actions/project
[info] loading settings for project sbt-github-actions from build.sbt,libraries.sbt ...
[info] looking for workflow definition in /home/runner/work/sbt-github-actions/sbt-github-actions/.github/workflows
[info] set current project to sbt-github-actions (in build file:/home/runner/work/sbt-github-actions/sbt-github-actions/)
[error] Switch failed: no subprojects list "2.12.[17](https://github.com/sbt/sbt-github-actions/actions/runs/3900050793/jobs/6660371077#step:8:18)" (or compatible version) in crossScalaVersions setting.
[error] If you want to force it regardless, call ++ 2.12.17!

Error: Process completed with exit code 1.
```

**Comment by mdedetrich** [comment]
@fthomas PR created at https://github.com/scala-steward-org/scala-steward/pull/2914

**Comment by mdedetrich** [comment]
@eed3si9n Regarding JDK version, this can be changed by updating https://github.com/sbt/sbt-github-actions/blob/main/build.sbt#L25. I can setup a PR, what JDK version should we be using? I noticed that its set to use GraalVM/JDK 11, I suppose using standard JVM JDK 17 is best bet to be safe?

**Comment by mdedetrich** [comment]
PR for updating to JDK 17 made at https://github.com/sbt/sbt-github-actions/pull/135

**Comment by mdedetrich** [comment]
So the above PR is passing and ready, I am not sure if upgrading to JDK 17 will fix the problem but nonetheless its the latest stable version so it should have been updated anyways. Once https://github.com/sbt/sbt-github-actions/pull/136 is merged I will also do another PR that will update the default Scala version to the latest as well

**Comment by eed3si9n** [comment]
Snapshot publishing worked - https://oss.sonatype.org/content/repositories/snapshots/com/github/sbt/sbt-github-actions_2.12_1.0/0.14.2-SNAPSHOT/

Now all you need is push a tag to make actual publish.

**Comment by mdedetrich** [comment]
Perfect, thanks for pushing this. I will make a PR tomorrow to update Scala version and then I will trigger a release
